### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by mikeshort10**

### CODE REVIEW

File: fuzz/Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ rust-version = "1.56.1"
 cargo-fuzz = true
```
Comment (xansec) [inline fuzz/Cargo.toml:11]: Are you using this target? If not, should it be omitted?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzz/Cargo.toml (hunk 1 above)
Comment (mikeshort10) [reply fuzz/Cargo.toml:11]: @xansec Not needed. I removed it.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**